### ISSUE TITLE
Crash after updating to latest version

### ISSUE THREAD

**Issue by NorthernMan54** [NorthernMan54/homebridge-alexa#198]
Help me!
After update your plugin:
```
/usr/lib/node_modules/homebridge-alexa/node_modules/hap-node-client/HAPNodeJSClient.js:107
      debug("HAP Device discovered", instance.txt.md, instance.addresses);
                                                  ^
TypeError: Cannot read property 'md' of undefined
    at HAPNodeJSClient.<anonymous> (/usr/lib/node_modules/homebridge-alexa/node_modules/hap-node-client/HAPNodeJSClient.js:107:51)
    at emitOne (events.js:116:13)
    at Browser.emit (events.js:211:7)
    at Browser._addService (/usr/lib/node_modules/homebridge-alexa/node_modules/bonjour/lib/browser.js:109:8)
    at /usr/lib/node_modules/homebridge-alexa/node_modules/bonjour/lib/browser.js:86:14
    at Array.forEach (<anonymous>)
    at /usr/lib/node_modules/homebridge-alexa/node_modules/bonjour/lib/browser.js:84:15
    at Array.forEach (<anonymous>)
    at EventEmitter._onresponse (/usr/lib/node_modules/homebridge-alexa/node_modules/bonjour/lib/browser.js:76:26)
    at emitTwo (events.js:126:13)
    at EventEmitter.emit (events.js:214:7)
    at Socket.<anonymous> (/usr/lib/node_modules/homebridge-alexa/node_modules/multicast-dns/index.js:49:43)
    at emitTwo (events.js:126:13)
    at Socket.emit (events.js:214:7)
    at UDP.onMessage (dgram.js:659:8)
```

Thank you

**Comment by NorthernMan54** [comment]
This was resolved with version v0.3.6

**Comment by giannimp** [comment]
It’s running! Thank you
